src/components/buttons/StandardButton.jsx

```
@@ -0,0 +1,35 @@
+function StandardButton({ label, onClick = null, id = null, type = null, className = "", variant = "primary", faIcon = "", tooltip = "", displayIconAsSuffix = false, size = "btn-default-size", status = "btn-enabled" }) {
+    return (
+        <button id={id}
+                type={type}
+                className={`standard-button btn text-3 btn-${variant} ${className} ${status} ${size}`}
+                data-tooltip={tooltip}
+                onClick={onClick}>
+            {!displayIconAsSuffix && faIcon && (
+                <i className={`${faIcon} me-2`}/>
+            )}
+
+            <span dangerouslySetInnerHTML={{__html: label}}/>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)